ros-ocp/templates/deployment-ingress.yaml

```diff
@@ -38,7 +39,19 @@ spec:
                 sleep 5
               done
               echo "MinIO is ready"
-              echo "MinIO is ready"
+        {{- else }}
+        - name: wait-for-odf
```

Comment:
why not keep the same container and just change the command? Instead of having `wait-for-minio` and `wait-for-odf`. Once you start adding other types you'll end up with many more potential containers.. it's just for simplicity that the container that waits for the storage to be ready is always there with the same name